packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -32,11 +32,22 @@ import { InputType } from './Input'
 import { Model } from './Model'
 import { PrismaClientClass } from './PrismaClient'
 
+type RuntimeName =
+  | 'binary'
+  | 'library'
+  | 'wasm'
+  | 'edge'
+  | 'edge-esm'
+  | 'index-browser'
+  | 'react-native'
+  | 'client'
+  | (string & {}) // workaround to also allow other strings while keeping auto-complete intact
```

Comment:
ℹ️  I dropped a similar workaround that was previously used in a PR yesterday as it was violating linters. I think this should work better now. 🤞